### DISCUSSION TITLE
Icecast YP Directory

### DISCUSSION THREAD

**Discussion by cheleentano** [AzuraCast/AzuraCast#6513] [Q&A]
Hi guys, everything shows fine in the internet-radio catalog, but the field where the site address is indicated is empty, although the link is specified in the settings. Is everyone like this?
![c8ae9801578e108246137fd38f6467e7](https://github.com/AzuraCast/AzuraCast/assets/50516099/913e1a21-888d-479a-bb97-a6c56995fe6a)

**Comment by Moonbase59** [comment]
I _guess_ you have to be registered and "verified" with them to have your URL shown.

**Reply by cheleentano** [reply]
Registration is not required for icecast. Take for example this radio https://stream.flowly-nature.com/ They are all right.

**Reply by Moonbase59** [reply]
If my assumtion isn’t the case, maybe there _is_ some subtle difference between Icecast 2.4.4 (what Flowly Nature uses) and Icecast 2.4.0-kh22 (what we use). You could ask Karl Heyes upstream, he’s very helpful and might know.
